### PR TITLE
adding info on initiate login uri

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** adding info on initiate login uri
- **Is this PR related to a Monolith release?** no

### Resolves:

* [OKTA-381015](https://oktainc.atlassian.net/browse/OKTA-381015)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/create-okta-application/index.md [hunk 1]
```diff
@@ -13,5 +13,6 @@ Before you can sign a user in, you need to create an Okta app integration that r
 1. Enter a name for your app integration (or leave the default value).
 1. Enter your callback route for the **Login redirect URI**. This is the full redirect URI that you defined in the <GuideLink link="../define-callback/">previous step</GuideLink> (for example, `com.okta.example:/login`).
 1. Enter your callback route for the **Logout redirect URI**. This is the full redirect URI that you defined in the <GuideLink link="../define-callback/">previous step</GuideLink> (for example, `com.okta.example:/logout`).
+1. Include a URI in the **Initiate login URI** box to have Okta initiate the sign-in flow by redirecting to this endpoint (for example, `https://example:0000.com/login`). This triggers the client to send an authorize request. This URI is also used when users reset their passwords and need to be redirected back to your app.
```
Comment (yuliu-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/create-okta-application/index.md:16]: It should not include mobile, Okta cannot initiate a request to mobile phone.

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/create-okta-application/index.md [hunk 2]
```diff
@@ -13,12 +13,11 @@ Before you can sign a user in, you need to create an Okta app integration that r
     > **Note:** It is important to choose the appropriate application type for apps which are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients are not designed to have, hence breaking the sign-in or sign-out flow.
 1. Enter a name for your app integration (or leave the default value).
 1. Enter values for the **Login redirect URI**. This is the callback from <GuideLink link="../define-callback/">Define a callback route</GuideLink>. Add values for local development (for example, `http://localhost:8080/login/callback` or `http://localhost:8080/authentication/login-callback` for Blazor) and production (for example, `https://app.example.com/login/callback`).
-
+1. Include a URI in the **Initiate login URI** box to have Okta initiate the sign-in flow by redirecting to this endpoint (for example, `https://example:0000.com/login`). This triggers the client to send an authorize request. This URI is also used when users reset their passwords and need to be redirected back to your app.
```
Comment (yuliu-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-spa/create-okta-application/index.md:16]: Same as above, for a spa app, Okta cannot reach to the app if app itself is not running in the browser. It has to the the user launch browser and start the flow.

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/create-okta-application/index.md [hunk 3]
```diff
@@ -13,13 +13,11 @@ Before you can sign a user in, you need to create an Okta app integration that r
     > **Note:** It is important to choose the appropriate application type for apps that are public clients. Failing to do so may result in Okta API endpoints attempting to verify an app's client secret, which public clients aren't designed to have, hence breaking the sign-in or sign-out flow.
 1. Enter a name for your app integration (or leave the default value).
 1. Enter values for the **Login redirect URI**. This is the callback described in <GuideLink link="../define-callback">Understand the callback route</GuideLink>. Add values for local development (for example, `http://localhost:8080/authorization-code/callback`) and production (for example, `https://app.example.com/authorization-code/callback`).
-
+1. Include a URI in the **Initiate login URI** box to have Okta initiate the sign-in flow by redirecting to this endpoint (for example, `https://example:0000.com/login`). This triggers the client to send an authorize request. This URI is also used when users reset their passwords and need to be redirected back to your app.
```
Comment (yuliu-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/create-okta-application/index.md:16]: Why reset password will redirect to the client, which client reset password will redirect to?

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/create-okta-application/index.md [hunk 4]
```diff
@@ -13,13 +13,11 @@ Before you can sign a user in, you need to create an Okta app integration that r
...
 <!-- removed step 3 until the base uri is returned to the app wizard ui in feb 2021 -->
-<!--
-3. Add the **Base URI** of your application during local development, such as `http://localhost:3000`. Also, add any base URIs where your application runs in production, such as `https://app.example.com`.
+<!-- 3. Add the **Base URI** of your application during local development, such as `http://localhost:3000`. Also, add any base URIs where your application runs in production, such as `https://app.example.com`.
 -->
-
-7. Click **Save** to finish creating the Okta app integration.
+1. Click **Save** to finish creating the Okta app integration.
```
Comment (yuliu-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-into-web-app/create-okta-application/index.md:20]: Just curious, what is `1.` and `7.`? It seems they are not sequence number.

Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-into-web-app/create-okta-application/index.md:20]: there's a line break above step 7 - so it breaks the auto-numbering that markdown does. But, after 7, I can go back to 1., 1., 1., and so on.

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/create-okta-application/index.md (hunk 3 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-into-web-app/create-okta-application/index.md:16]: Please see: https://oktainc.atlassian.net/browse/OKTA-381015 
and
https://github.com/okta/okta-aspnet/issues/170

These are the reason I added that sentence on resetting passwords......

File: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/create-okta-application/index.md (hunk 1 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/create-okta-application/index.md:16]: @yuliu-okta and @venkatviswanathan-okta 

Should the tooltip text for this field in the Admin Console specify this field is for Web Apps only then? This is the tooltip text that Venkat and you approved:

Include a URI to have Okta initiate the sign-in flow by redirecting to this endpoint. This triggers the client to send an authorize request.

Should we add that it's for browser-based apps only?

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/create-okta-application/index.md (hunk 2 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-into-spa/create-okta-application/index.md:16]: @yuliu-okta and @venkatviswanathan-okta 

Should the tooltip text for this field in the Admin Console specify this field is for Web Apps only then? This is the tooltip text that Venkat and you approved:

Include a URI to have Okta initiate the sign-in flow by redirecting to this endpoint. This triggers the client to send an authorize request.

Should we add that it's for browser-based apps only?

File: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/create-okta-application/index.md (hunk 3 above)
Comment (yuliu-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-into-web-app/create-okta-application/index.md:16]: Ok, so it is user reset the password during signing into the client, then after reset, okta will redirect to the initiate_login_uri of the client.

File: packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/create-okta-application/index.md (hunk 1 above)
Comment (venkatviswanathan-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app/create-okta-application/index.md:16]: @yuliu-okta Do we use this with Native apps today? I see the configuration available for native apps as well.

File: packages/@okta/vuepress-site/docs/guides/sign-into-spa/create-okta-application/index.md (hunk 2 above)
Comment (venkatviswanathan-okta) [reply packages/@okta/vuepress-site/docs/guides/sign-into-spa/create-okta-application/index.md:16]: SPA == browser-based. You mean web app?

**Review by venkatviswanathan-okta — APPROVED** [review]

**Review by yuliu-okta — APPROVED** [review]

**Review by laura-rodriguez — APPROVED** [review]
LGTM, thanks Susan!

**Pull request merged**